### PR TITLE
Charlotte & Jonathan can make comments

### PR DESCRIPTION
**Pull request by CjMoore**
- we can make comments appear

### CODE REVIEW

File: source/addComment.js [hunk 1]
```diff
@@ -1,3 +1,11 @@
...
+  event.preventDefault();
+
+  var formInput = $('input[name=comment]').val()
+
+  $('#comment_list').append("<li>"+ formInput +"</li>")
+}
+
 $(document).ready(function(){
-  // add your code here
+  $("#add-comment").submit(doSomething)
```
Comment (case-eee) [inline source/addComment.js]: again, i'd name this more appropriately rather than just `doSomething`

File: source/addComment.js [hunk 2]
```diff
@@ -1,3 +1,11 @@
+function doSomething(){
+  event.preventDefault();
+
+  var formInput = $('input[name=comment]').val()
```
Comment (case-eee) [inline source/addComment.js]: 👍

**Review by case-eee — COMMENTED** [review]
@CjMoore @JonathanSR this is a great start! you'll want to add the functionality for a user to enter the author and also a user to not be able to submit without entering comment text 👍

**Comment by CjMoore** [comment]
@case-eee We changed the function name and added functionality for user inputting author.

**Comment by case-eee** [comment]
this looks AWESOME @CjMoore @JonathanSR - good work!

**Pull request closed without merging**